java-library/pom.xml

```diff
@@ -4,7 +4,7 @@
 
   <groupId>com.microsoft.azure.functions</groupId>
   <artifactId>azure-functions-java-library-sql</artifactId>
-  <version>2.1.0-preview</version> <!-- Update the dependency version in samples-java/pom.xml and test-java/pom.xml if this version is updated. -->
+  <version>2.1.0</version> <!-- Update the dependency version in samples-java/pom.xml and test-java/pom.xml if this version is updated. -->
```

Comment:
Will update version in samples once the library is released.

Reply:
Do we have an issue for tracking this?

Reply:
Created issue: https://github.com/Azure/azure-functions-sql-extension/issues/948 to track.